ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ccj/CountyCourtJudgmentMapperTest.java

```diff
@@ -63,9 +67,11 @@ public void shouldMapCountyCourtJudgmentWithPaymentOptionImmediatelyToCCD() {
     public void shouldMapCountyCourtJudgmentWithPaymentOptionSpecifiedDateToCCD() {
         //given
         final CountyCourtJudgment countyCourtJudgment = SampleCountyCourtJudgment.builder()
+            .ccjType(CountyCourtJudgmentType.ADMISSIONS)
             .paymentOption(PaymentOption.BY_SPECIFIED_DATE)
             .payBySetDate(now())
             .statementOfTruth(new StatementOfTruth("Tester", "Unit Test"))
+            .ccjType(CountyCourtJudgmentType.DEFAULT)
```

Comment:
Either need this line or Line 70, but not both.

Reply:
removed